src/Randomizer.SMZ3.Tracking/Services/ItemService.cs

```diff
@@ -86,14 +86,21 @@ public ItemService(ItemConfig items, RewardConfig rewards, IWorldAccessor world)
         /// tracked at least once; otherwise, <see langword="false"/>.
         /// </returns>
         public virtual bool IsTracked(ItemType itemType)
-            => AllItems().Any(x => x.Type == itemType && x.State.TrackingState > 0);
+            => LocalPlayersItems().Any(x => x.Type == itemType && x.State.TrackingState > 0);
 
         /// <summary>
         /// Enumerates all items that can be tracked.
         /// </summary>
         /// <returns>A collection of items.</returns>
         public IEnumerable<Item> AllItems() // I really want to discourage this, but necessary for now
-            => _world.World.AllItems;
+            => _world.Worlds.SelectMany(x => x.AllItems);
+
+        /// <summary>
+        /// Enumerates all items that can be tracked.
```

Comment:
Clarify in the description that this is only for the local player, and/or update the other one to mention it includes items from other players